components/script/dom/html/htmlelement.rs

```diff
@@ -1268,10 +1285,34 @@ impl VirtualMethods for HTMLElement {
         }
     }
 
+    fn attribute_affects_presentational_hints(&self, attr: &Attr) -> bool {
+        if attr.local_name() == &local_name!("bgcolor") {
```

Comment:
Why `bgcolor` didn't mentioned in PR description/title and you added it as hint for any HTML element, is it right?